docs/en/02_Developer_Guides/09_Security/02_Permissions.md

```diff
@@ -105,6 +105,17 @@ if (SilverStripe\Security\Permission::checkMember($member, 'CMS_ACCESS')) {
 
 Internally, this checks that the user has any of the defined `CMS_ACCESS_*` permissions.
 
+For example if your Namespace is Company\Website then the canView() method of your DataObject will look like this for a ModelAdmin sub class named NewsAdmin:
+
+```
+	public function canView($member = null)
+	{
+		return Permission::check('CMS_ACCESS_Company\Website\NewsAdmin', 'any', $member);
```

Comment:
I just ran a test locally. I didn't get the fully qualified name of my Model Admin. It cameback as `CMS_ACCESS_NewsAdmin` for me.